app/assets/stylesheets/provider/admin/_dashboard.scss

```diff
@@ -30,6 +30,7 @@
     color: $font-color;
     min-width: 10%;
     float: left;
+    max-width: 40%;
```

Comment:
I think 40% is a lot. I'm using `min-width: 10%`;

Reply:
However 10% is ineffective:
<img width="1101" alt="screen shot 2018-11-05 at 10 03 52" src="https://user-images.githubusercontent.com/11672286/47988298-2010c200-e0e2-11e8-8f72-1c30d1152d01.png">

At least I'd go for 25%, it has a better balance:
<img width="1089" alt="screen shot 2018-11-05 at 10 04 51" src="https://user-images.githubusercontent.com/11672286/47988425-77af2d80-e0e2-11e8-8bd4-89675217ec23.png">

@thomasmaas should have the final word though.

Reply:
Good point, I haven't tested such a loooong title

Reply:
@damianpm I think we should use ellipsis for long titles here. With full name in title attribute

Reply:
@thomasmaas @josemigallas 
max-width: 20% and text ellipsis:
![ellipsis](https://user-images.githubusercontent.com/13486237/47990268-661c5480-e0e7-11e8-8d10-1a1ed76c0d33.png)